ci.cjs

```diff
@@ -95,7 +95,7 @@ async function checkConsistency(file, versionsMap) {
 
   for (const [name, versionInDep] of versionsMap) {
     const check = (x, version) => {
-      if (version.includes("*")) {
+      if (version.includes("*") || version.includes("^")) {
```

Comment:
What does this mean? What does this do?

Reply:
This file bumps version in ci to support e2e testing. Here i added support for workspace symbol.